packages/bull-vault/src/LeverageBull.sol

```diff
@@ -125,16 +125,16 @@ contract LeverageBull is Ownable {
         uint256 _squeethInCrab,
         uint256 _crabTotalSupply
     ) internal returns (uint256, uint256, uint256) {
-        (uint256 ethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
+        (uint256 wethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
```

Comment:
Maybe `_calcLeverageEthUsdc` to `_calcLeverageWethUsdc`

Reply:
happy to change this one as well